src/lib/dnssd/minimal_mdns/Server.cpp

```diff
@@ -188,6 +240,10 @@ CHIP_ERROR ServerBase::Listen(chip::Inet::InetLayer * inetLayer, ListenIterator
         {
             endpointIndex++;
         }
+
+        ReturnErrorOnFailure(inetLayer->NewUDPEndPoint(&info->query_udp));
+        ReturnErrorOnFailure(info->query_udp->Bind(addressType, chip::Inet::IPAddress::Any, kPickRandomBindPort, interfaceId));
```

Comment:
The name kPickRandomBindPort feels like adherence to the no-magic-numbers rule at the detriment of clarity and readability.  Using a name for this constexpr that makes it clear we are binding to the reserved 0th port, or even a 0 literal, would be clearer in my opinion.  The fact port 0 has a special use for this purpose is a ubiquitous and important concept.  

With this named as it is, I had to look back at the definition of kPickRandomBindPort to make sure we aren't using some other special value (which would be a poor design).  This is purely a nit.  But in my opinion, this is too much obfuscation.

Reply:
That is a fair point, used 0 directly and updated the comment a bit to explain what 0 does in case anyone is unclear (although it should really be clear).